docs/source/bali/index.rst

```diff
@@ -0,0 +1,83 @@
+====
+Bali
+====
+
+Bali is the planned third release of the OpenSDS project. It is currently under
+development with a planned release date of the week of December 17, 2018.
+
+Bali Schedule
+=============
+
+.. datatemplate::
+   :source: schedule.yaml
+   :template: schedule_table.tmpl
+
+.. note::
+
+   These dates are based on the current plan and are subject to change.
+
+Bali Milestones
+===============
+
+.. _b-1:
+
+Milestone 1
+-----------
+
+The first Bali milestone will be the week of August 6.
+
+This first milestone serves as a checkpoint on the progress so far and as a
+good point to make sure our release processes are in order.
+
+.. _b-2:
+
+Milestone 2
+-----------
+
+The second Bali milestone will be the week of September 24.
+
+No new features may be proposed for the Bali release without the review and
+approval of the Technical Steering Committee (TSC).
+
+.. _b-3:
+
+Milestone 3
+-----------
+
+The third Bali milestone will be the week of November 12.
+
+All new feature development should be complete by the third milestone. It will
+be at the discretion of each project lead to allow any code changes other than
+bug fixes after this point. Leads should make these decisions based on the risk
+introduced by the code change and based on the importance of the feature. The
+TSC should be consulted if the priorities and risks are unclear.
+
+.. _b-rc:
+
+Release Candidate
+-----------------
+
+A release candidate will be done the week of December 3.
+
+The RC code should be the focus of all test efforts leading up to the final
+release. Only critical bug fixes should be allowed to be merged after this
+point to ensure the final release is in good shape.
+
+.. _b-release:
+
+Bali Release
+------------
+
+The final Bali release is planned for the week of December 17.
+
+Bali Events
+===========
+
+.. _b-ms:
+
+Mini-Summit
+-----------
+
+An OpenSDS Mini-Summit, geared towards both users and developers, is planned to
+be colocated at the Linux Foundation Open Source Summit EU, November 13-15 in
+Edinburgh, Scotland.
```

Comment:
Have you checked with Steven on publishing this?  I'm not sure if we've made a final decision on this yet.

Reply:
It is still being finalized, but I wanted to include it here being optimistic. We should know soon. Steven is added as a reviewer on this PR as well.

If it ends up falling through we can quickly drop it from the schedule.